### PR TITLE
Rename volume scripts

### PR DESCRIPTION
**Pull request by arnaudbore**
scil_apply_transform_to_image.py - > scil_volume_apply_transfo.py
 scil_count_non_zero_voxels.py ->  scil_volume_count_non_zero_voxels.py
 scil_crop_volume.py ->  scil_volume_crop_.py
 scil_flip_volume.py ->  scil_volume_flip.py
 scil_image_math.py -> scil_volume_math.py
 scil_remove_outliers_ransac.py ->  scil_volume_remove_outliers_ransac.py
 scil_resample_volume.py ->  scil_volume_resample.py
 scil_reshape_to_reference.py ->  scil_volume_reshape_to_reference.py
 scil_snr_in_roi.py ->  scil_dwi_compute_snr.py

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/legacy/scil_reshape_to_reference.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from scilpy.io.deprecator import deprecate_script
+from scripts.scil_volume_reshape_to_reference import main as new_main
+
+
+DEPRECATION_MSG = """
+This script has been renamed scil_image_reshape_to_reference.py. Please change
```
Comment (karanphil) [inline scripts/legacy/scil_reshape_to_reference.py:9]: scil_volume*

**Review by karanphil — COMMENTED** [review]
Spotted a small error in a legacy script. Not sure if `volume `should be replaced with `image` to have like `scil_image_flip_volume.py` or create the `volume` module instead of the `volume_operations` in the `image` module. Also, I understand the name `scil_dwi_compute_snr.py`, but it imports nothing from `dwi`.

**Comment by EmmaRenauld** [comment]
> Spotted a small error in a legacy script. Not sure if `volume `should be replaced with `image` to have like `scil_image_flip_volume.py` or create the `volume` module instead of the `volume_operations` in the `image` module. 

I think we prefer volume everywhere. It's clearer than image, which could be a picture. Do we want to change the image module to volume?

> Also, I understand the name `scil_dwi_compute_snr.py`, but it imports nothing from `dwi`.

Indeed, the compute_snr function should go in dwi. It is now in volume_operations. But it takes as input the dwi, bval, bvec, so it is not appliable to any volume.

File: scripts/legacy/scil_split_volume_by_ids.py [hunk 2]
```diff
@@ -2,18 +2,18 @@
 # -*- coding: utf-8 -*-
 
 from scilpy.io.deprecator import deprecate_script
-from scripts.scil_tracking_local import main as new_main
```
Comment (EmmaRenauld) [inline scripts/legacy/scil_split_volume_by_ids.py:5]: I did this mistake a bit everywhere. Corrected in PR #832

File: scripts/legacy/scil_split_volume_by_ids.py [hunk 3]
```diff
@@ -2,18 +2,18 @@
...
+This script has been renamed scil_labels_split_volume_by_ids.py.
 Now, all our scripts using labels start with scil_labels_...!
 
 Please change your existing pipelines accordingly.
 """
 
 
-@deprecate_script("scil_split_volume_by_is.py", DEPRECATION_MSG, '1.7.0')
+@deprecate_script("scil_split_volume_by_ids.py", DEPRECATION_MSG, '1.7.0')
 def main():
```
Comment (EmmaRenauld) [inline scripts/legacy/scil_split_volume_by_ids.py:17]: Thanks.

**Review by EmmaRenauld — APPROVED** [review]

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by frheault** [comment]
Has anyone ever used scil_volume_remove_outliers_ransac?

**Pull request merged**